AstSemantics.md

```diff
@@ -286,10 +255,8 @@ Each function has a *signature*, which consists of:
   * Return types, which are a sequence of local types
   * Argument types, which are a sequence of local types
 
-Note that WebAssembly itself does not support variable-length argument lists
-(aka varargs). C and C++ compilers are expected to implement this functionality
-by storing arguments in a buffer in linear memory and passing a pointer to the
-buffer.
+WebAssembly doesn't support variable-length argument lists (aka varargs), they
+are instead supported through explicit accesses to linear memory.
```

Comment:
Please parenthesize or otherwise demark the note about using linear memory to implement varargs in a way that makes it clear that this is just informative content, and not something that WebAssembly's own semantics do themselves.